### ISSUE TITLE
case statement in inline_template false warning

### ISSUE THREAD

**Issue by dalen** [rodjek/puppet-lint#117]
I have a manifest that converts the memorysize fact and converts it to bytes to be able to do some comparisons on it. dust_bunny started complaining about case statements and resources without quoted titles. Here's the manifest snippet:

```
  # facter is borked and returns memory size as a string so this is ugly
  $mem = inline_template('<%
        mem,unit = scope.lookupvar(\'::memorysize\').split
        mem = mem.to_f
        # Normalize mem to bytes
        case unit
            when nil:  mem *= (1<<0)
            when \'kB\': mem *= (1<<10)
            when \'MB\': mem *= (1<<20)
            when \'GB\': mem *= (1<<30)
            when \'TB\': mem *= (1<<40)
        end
        %><%= mem.to_i %>')
```
